documentation/tableOfContents.ts

```diff
@@ -14,6 +14,13 @@ export const tableOfContents: Record<string, string[]> = {
 		(await import("./getting-started.md?raw")).default,
 		(await import("./why-inlang.md?raw")).default,
 	],
+	SDK: [
```

Comment:
I moved those files into subdirectories. I found it hard to find the files and see what belongs where.
Also the links should be accessible in a hierarchy `/sdk/general` instead of `/general`

Reply:
Alright, let's keep it that way 👍